scripts/gitter-lockdown.coffee

```diff
@@ -0,0 +1,76 @@
+# Description:
+#   When enabled in a Gitter room, should remove all new messages
+#
+# Commands:
+#   hubot lockdown - Adds the room to a list of rooms to lock down.
+#   .+ - When a room is locked down, removes any messages it hears that aren't from a mod.
+#
+# Author:
+#   jpruskin
+
+module.exports = (robot) ->
+  # Helpful, thanks to https://doatt.com/2015/02/19/the-hubot-msg-object/index.html
+  # robot.respond /debug/, (msg) ->
+  #   msg.finish()
+  #   console.log Object(msg)
+
+  moderators = [
+    '6064213c6da037398478addf'  # jpruskin
+    '5319033d5e986b0712efd300'  # ferventcoder
+    '530cd1a05e986b0712efb4ce'  # gep13
+  ]
+
+  if robot.brain.get('lockedRooms') == null
+    robot.brain.set 'lockedRooms', [
+      # '52a5ea24ed5ab0b3bf05002d'  # chocolatey
+      # '54c5e7f6db8155e6700f14bf'  # chocolatey/chocobot-gitter
+      '53190d1b5e986b0712efd33c'  # chocolatey/chocolatey.org
+      '54d1cb5fdb8155e6700f6c9c'  # chocolatey/chocolatey-oneget
+      '531444a55e986b0712efc533'  # chocolatey/chocolatey
+      '530cd0b45e986b0712efb4cb'  # chocolatey/ChocolateyGUI
+      '54bc1bf1db8155e6700ecc71'  # chocolatey/choco
+      '58b5ba05d73408ce4f4d818c'  # chocolatey/chocolatey-coreteampackages
```

Comment:
```suggestion
      '58b5ba05d73408ce4f4d818c'  # chocolatey/chocolatey-coreteampackages
      '54e9026715522ed4b3dc4cb6' # chocolatey/puppet-chocolatey
      '58f69dc0d73408ce4f595206' # chocolatey/cChoco
      '5e2f484fd73408ce4fd7ed4b' # chocolatey-community/community
```

Reply:
```suggestion
      '58b5ba05d73408ce4f4d818c'  # chocolatey/chocolatey-coreteampackages
      '54e9026715522ed4b3dc4cb6'  # chocolatey/puppet-chocolatey
      '58f69dc0d73408ce4f595206'  # chocolatey/cChoco
      '5e2f484fd73408ce4fd7ed4b'  # chocolatey-community/community
```

Reply:
Added choco-bot to those new rooms - good catch, thank you!

Reply:
Just a note to please squash the commits when merging this.